package.json

```diff
@@ -74,6 +74,8 @@
     "prepack": "yarn build && yarn build-tokens",
     "storybook": "start-storybook -p 6006",
     "build": "yarn build:prod",
+    "postinstall:addon": "yarn build:addon && yarn build-tokens",
```

Comment:
When will this hook be fired? Aka. How is demosplan-ui consumed within addons?

Reply:
This hook needs to be fired when installing addons right before creating the addon components

Reply:
Does this need to be part of the installation process in demosplan-core then?

Reply:
Yes this was my intention so we can use addons right away after the installation is done

Reply:
Those hooks were dropped since the current config is now valid for our core applications and also for our addons. Unfortunately I could not get rid of umd yet but this is a starter to reduce the duplications of components in our addons.